index.js

```diff
@@ -15,14 +15,14 @@ function undici (url, opts) {
   return new Pool(url, opts)
 }
 
-undici.Pool = Pool
-undici.Client = Client
-undici.errors = errors
+module.exports = undici
 
-undici.Agent = Agent
-undici.request = request
-undici.stream = stream
-undici.pipeline = pipeline
-undici.setGlobalAgent = setGlobalAgent
+module.exports.Pool = Pool
```

Comment:
This fails on Node `12.18.0`, but works on `12.21` (and later), and I could figure out a  way to fix it.

Reply:
From the docs [here](https://nodejs.org/dist./v12.18.0/docs/api/esm.html#esm_import_statements) seems it is not supported on that version, maybe we should skip it?

Reply:
It seems to be fixed in `12.20`:

https://nodejs.org/dist./v12.20.0/docs/api/esm.html#esm_import_statements

Reply:
Yes, this was added in v12.20